utility_network/display-utility-associations/src/main/resources/display_utility_associations/style.css

```
@@ -0,0 +1,4 @@
+.panel-region .label {
+  -fx-text-fill: white;
+}
```

If you decide to use the grid-pane suggestion from above, here's how to get it to customized a little with css:

```suggestion
.grid-pane {
  -fx-padding: 10;
  -fx-background-color: rgba(255,255,255,0.5);
  -fx-max-height: 80;
  -fx-max-width: 100;
  -fx-vgap: 20;
  -fx-alignment: center;
}
```